src/editor/UI/Commandline.re

```diff
@@ -0,0 +1,51 @@
+open Revery.UI;
+open Revery.Core;
+open Oni_Core;
+
+let component = React.component("commandline");
+
+let make = (~command: Types.Commandline.t, ~theme: Theme.t) => {
+  component((_slots: React.Hooks.empty) =>
+    command.show
+      ? <View
+          style=Style.[
+            position(`Relative),
+            top(0),
+            right(0),
+            left(0),
+            bottom(0),
+          ]>
+          <View
+            style=Style.[
+              width(400),
+              height(40),
+              top(-30),
+              left(50),
+              backgroundColor(theme.editorBackground),
+              boxShadow(
+                ~xOffset=-15.,
+                ~yOffset=5.,
+                ~blurRadius=30.,
+                ~spreadRadius=5.,
+                ~color=Color.rgba(0., 0., 0., 0.2),
+              ),
+            ]>
+            <Text
+              style=Style.[
+                fontFamily("FiraCode-Regular.ttf"),
+                marginTop(10),
+                marginLeft(10),
+                fontSize(20),
+                color(Colors.white),
+              ]
+              text={command.firstC ++ command.content}
+            />
+          </View>
+        </View>
+      : <View />
```

Comment:
@bryphe suprised I haven't encountered this yet but how do we return an empty value from a component i.e. render nothing conditionally?

Reply:
Oh, good question! @wokalski might know a better way - but I think one possibility is to use `React.listToElement([])`. We could potentially alias `let empty = React.listToElement([])`, I suppose.

Reply:
Yup!

Reply:
The alias sounds like a good idea, I think `ReasonReact` has a `ReasonReact.empty` value that can be used should it be something we add to the project or to the `React` module itself? (seems best in the React module?)

Reply:
Cool! @Akin909 - would you be up for opening a PR on `brisk-reconciler` to add this?

I think the work would be adding a signature here: 
https://github.com/briskml/brisk-reconciler/blob/master/lib/Brisk_reconciler.rei

Like `let empty: syntheticElement;`

And then adding the implementation here: 
https://github.com/briskml/brisk-reconciler

Like `let empty = React.listToElement([]);`

That way we could get @wokalski 's feedback on it! We could also implement it in Revery's react namespace, but I think it seems like it would be useful across the board for brisk-reconciler based apps.